module/dice/ed-roll.mjs

```diff
@@ -324,9 +328,9 @@ export default class EdRoll extends Roll {
     templateData.step = this.options.step;
     templateData.target = this.options.target;
     templateData.testType = ED4E.testTypes[this.options.testType].label;
+    templateData.ruleOfOne = this.isRuleOfOne;
```

Comment:
Doesn't really matter, but why did that move up? Just out of curiosity :D

Reply:
because failure and success will be overwritten by rule of one. I added a line in isSuccess and isFailure to chat the rule of one